darts/tests/models/forecasting/test_local_forecasting_models.py

```diff
@@ -59,14 +51,10 @@
     (FFT(trend="poly"), 11.4),
     (NaiveSeasonal(), 32.4),
     (KalmanForecaster(dim_x=3), 17.0),
+    (LinearRegressionModel(lags=12), 11.0),
+    (RandomForest(lags=12, n_estimators=200, max_depth=3), 15.5),
```

Comment:
Isn't `n_estimators=200` a bit too much for a unit test? This doesn't take too long?

Reply:
I just took this from a few lines below, tbh. I can reduce it.